openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -164,7 +164,8 @@
         "BITMAP",
         "UINT",
         "BIT",
-        "MONEY"
+        "MONEY",
+        "INTEGER"
```

Comment:
Lets not add this datatype here, we should already have a datatype that we can put here